cli/src/main/resources/help/module.txt

```diff
@@ -49,7 +49,7 @@ ARGUMENTS
  --name          - (required) the name of the module to be added or removed.
                    NOTE: slot is not a part of the module name.
 
- --slot          - (optional) specifies a slot which should be created or
+ --slot          - (deprecated, optional) specifies a slot which should be created or
```

Comment:
I was under the impression that kind of name wasn't allowed but I think I misunderstood how the parser works.

But then this begs a question, if a user calls `module add --name=org.example.module --slot=test` and we detect version 1.9 then instead of downgrading to 1.5 we could make it work as if the user called `module add --name=org.example.module:test` and no downgrade is necessary

Reply:
@michpetrov , yes I thought at it too. I liked the idea but in some way it uses a deprecated concept (slot) to convey the new version information. The JBoss modules documentation: https://jboss-modules.github.io/jboss-modules/manual/#versions explains it well.
I think that we should clearly specify the behavior in all the different cases, a wildfly-proposals would be a nice way to capture the new command logic: https://github.com/wildfly/wildfly-proposals/